### PR TITLE
Add sandbox SSH alias

### PR DESCRIPTION
**Pull request by jakubno**
To improve DX you can now run only 
```bash
sbx-ssh <sandboxID>
```
and you will be directly connected to the sandbox.

### CODE REVIEW

**Comment by mlejva** [comment]
Is this a legit ssh now? I remember we had to fame it before

**Comment by jakubno** [comment]
It's inside the host. You can connect to client and then to sandbox, it's DX for e2b devs.

**Review by ValentaTomas — APPROVED** [review]

**Comment by 0div** [comment]
Thx for this! 🙏 FYI i built the equivalent as util func in the orchestrator go code, using it for tests for now.

File: packages/cluster/scripts/start-client.sh [hunk 1]
```diff
@@ -153,3 +153,11 @@ echo $overcommitment_hugepages >/proc/sys/vm/nr_overcommit_hugepages
 # These variables are passed in via Terraform template interpolation
 /opt/consul/bin/run-consul.sh --client --consul-token "${CONSUL_TOKEN}" --cluster-tag-name "${CLUSTER_TAG_NAME}" --enable-gossip-encryption --gossip-encryption-key "${CONSUL_GOSSIP_ENCRYPTION_KEY}" &
 /opt/nomad/bin/run-nomad.sh --client --consul-token "${CONSUL_TOKEN}" &
+
+# Add alias for ssh-ing to sbx
+echo '_sbx_ssh() {
+  local address=$(dig @localhost $1. A +short 2>/dev/null)
```
Comment (0div) [inline packages/cluster/scripts/start-client.sh:159]: on my host at least, localhost doesn't return the IP of a running instance but this does:
```
jonas@jtdevorch-client-fkl5:~/infra/packages/orchestrator$ dig @127.0.0.4 test-instance-1 A +short
192.168.2.197

jonas@jtdevorch-client-fkl5:~/infra/packages/orchestrator$ dig @localhost test-instance-1 A +short
<nada>
```

Comment (jakubno) [reply packages/cluster/scripts/start-client.sh:159]: looks like there must be something different, works for me on a clean deploy

Comment (0div) [reply packages/cluster/scripts/start-client.sh:159]: maybe worth noting: im looking for instances created via `make test`

**Pull request merged**